### PR TITLE
👷 Test configuration with no AWS access

### PR DESCRIPTION
**Pull request by falexwolf**
https://laminlabs.slack.com/archives/C04FPE8V01W/p1691442944937179?thread_ts=1691410353.797789&cid=C04FPE8V01W

### CODE REVIEW

**Comment by falexwolf** [comment]
I've constructed a dedicated test case! I'm not running the unit tests, here!

**Comment by falexwolf** [comment]
It's only the `noaws` job that's testing this setup and it has it's own test associated to it.

**Comment by Koncopd** [comment]
This fails for lndb-setup-ci, is it really properly configured to be publicly accessible? It gives access error on `file.exists()`, so i suppose it is not.

**Comment by Koncopd** [comment]
on lamindata i was able to stage a file, it wasn't giving me errors on read-only object access, so the problem here is probably in the bucket configuration.

**Comment by falexwolf** [comment]
Right, Sergei! Just noticing the same things on Slack: https://laminlabs.slack.com/archives/C04FPE8V01W/p1691442417881599?thread_ts=1691410353.797789&cid=C04FPE8V01W

**Comment by falexwolf** [comment]
You're right, I seem to be missing a step in the config

**Comment by Koncopd** [comment]
@sunnyosun might remember how to configure for public access, we had similar problems for some buckets used for bionty assets.

**Comment by Koncopd** [comment]
We really need to document such things somewhere i guess.

**Comment by falexwolf** [comment]
Yes, I fully agree - pasted the same notes on Slack!

**Comment by falexwolf** [comment]
Now it works! :D
```
tests/test_load_persistent_instance.py 🔶 did not find aws credentials, using anonymous
🔶 updating local SQLite & locking cloud SQLite (sync back & unlock: lamin close)
🔶 read-only access - did not access locker
🔶 read-only access - did not access locker
🌱 saved: User(id='DzTjkKse', handle='testuser1', email='testuser1@lamin.ai', name='Test User1', updated_at=2023-08-07 21:21:53)
🔶 did not find aws credentials, using anonymous
✅ loaded instance: laminapp-admin/persistent-instance-to-test-load
./opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/coverage/control.py:860: CoverageWarning: No data was collected. (no-data-collected)
  self._warn("No data was collected.", slug="no-data-collected")
/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/pytest_cov/plugin.py:298: CovReportWarning: Failed to generate report: No data to report.
  self.cov_controller.finish()
WARNING: Failed to generate report: No data to report.
```

File: noxfile.py [hunk 1]
```diff
@@ -43,7 +46,7 @@ def build(session: nox.Session, group: str, lamin_env: str):
     coverage_args = "--cov=lamindb_setup --cov-append --cov-report=term-missing"  # noqa
     if group.startswith("unit"):
         session.run(
-            *f"pytest -s {coverage_args} ./tests --ignore tests/hub".split(),
+            *f"pytest -s {coverage_args} ./tests/unit".split(),
```
Comment (falexwolf) [inline noxfile.py:49]: @bpenteado, please let me know if this is going to cause problems for the `lnhub-rest` tests in some form!

**Pull request merged**